File: model/rango_seccion.py
from utils.db import db
from dataclasses import dataclass

@dataclass
class Rango_seccion(db.Model):
    __tablename__='rango_seccion'
    id_rango_seccion = db.Column(db.Integer, primary_key=True, autoincrement=True)
    id_seccion = db.Column(db.Integer, db.ForeignKey('seccion.id_seccion'))
    minimo = db.Column(db.Integer)
    maximo = db.Column(db.Integer)
    diagnostico = db.Column(db.String(250))
    
    test = db.relationship('Seccion', backref='rango_seccion')

    def __init__(self,id_seccion,minimo,maximo,diagnostico):
        self.id_seccion= id_seccion
        self.minimo = minimo       
        self.maximo = maximo
        self.diagnostico = diagnostico

File: v2-microservicios/sisvita-app/realizar-test-service/src/main/java/com/realizar_test_service/realizar_test_service/model/model_temporal/RangoSeccion.java
package com.realizar_test_service.realizar_test_service.model.model_temporal;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

import java.util.List;

public class RangoSeccion {

    private Integer idRangoSeccion;
    private Integer idSeccion; // Referencia a la Seccion
    private Integer minimo;
    private Integer maximo;
    private String diagnostico;

    // Constructor, getters y setters

    public RangoSeccion() {
    }

    public RangoSeccion(Integer idRangoSeccion, Integer idSeccion, Integer minimo, Integer maximo, String diagnostico) {
        this.idRangoSeccion = idRangoSeccion;
        this.idSeccion = idSeccion;
        this.minimo = minimo;
        this.maximo = maximo;
        this.diagnostico = diagnostico;
    }

    public Integer getIdRangoSeccion() {
        return idRangoSeccion;
    }

    public void setIdRangoSeccion(Integer idRangoSeccion) {
        this.idRangoSeccion = idRangoSeccion;
    }

    public Integer getIdSeccion() {
        return idSeccion;
    }

    public void setIdSeccion(Integer idSeccion) {
        this.idSeccion = idSeccion;
    }

    public Integer getMinimo() {
        return minimo;
    }

    public void setMinimo(Integer minimo) {
        this.minimo = minimo;
    }

    public Integer getMaximo() {
        return maximo;
    }

    public void setMaximo(Integer maximo) {
        this.maximo = maximo;
    }

    public String getDiagnostico() {
        return diagnostico;
    }

    public void setDiagnostico(String diagnostico) {
        this.diagnostico = diagnostico;
    }

}
